src/Components/AuthLogin/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./styles.css";
+import { Link } from "react-router-dom";
+
+function AuthLogin() {
+  return (
+    <div class="container-login flex-c">
+      <h1>Login</h1>
+
+      <div class="email">
+        <label for="email-id"> Email address</label>
```

Comment:
Use `htmlFor` instead of `for`

Reply:
Thank you for pointing it out i will fix that